pkg/console/subresource/configmap/configmap.go

```diff
@@ -40,54 +36,62 @@ func getApiUrl(infrastructureConfig *configv1.Infrastructure) string {
 	return ""
 }
 
-// DefaultConfigMap returns
-// - a new configmap,
-// - a bool indicating if config was merged (unsupportedConfigOverrides)
-// - an error
-func DefaultConfigMap(operatorConfig *operatorv1.Console, consoleConfig *configv1.Console, managedConfig *corev1.ConfigMap, infrastructureConfig *configv1.Infrastructure, rt *routev1.Route) (*corev1.ConfigMap, bool, error) {
-
-	// Build a default config that can be overwritten from other sources
-	host := rt.Spec.Host
-	apiServerURL := getApiUrl(infrastructureConfig)
-	defaultProviders := operatorv1.ConsoleProviders{}
-	defaultConfig := NewYamlConfig(host, defaultLogoutURL, DEFAULT_BRAND, DEFAULT_DOC_URL, apiServerURL, defaultProviders)
-	// Get console config from the openshift-config-managed namespace
+func statusPageId(operatorConfig *operatorv1.Console) string {
+	if operatorConfig.Spec.Providers.Statuspage != nil {
+		return operatorConfig.Spec.Providers.Statuspage.PageID
+	}
+	return ""
+}
+
+func DefaultConfigMap(
+	operatorConfig *operatorv1.Console,
+	consoleConfig *configv1.Console,
+	managedConfig *corev1.ConfigMap,
+	infrastructureConfig *configv1.Infrastructure,
+	rt *routev1.Route) (consoleConfigmap *corev1.ConfigMap, unsupportedConfigMerged bool, err error) {
+
+	b := &ConsoleYAMLConfigBuilder{}
```

Comment:
nit: would rather have named variables.. here and below :)
```suggestion
	defaultBuilder := &ConsoleYAMLConfigBuilder{}
```

Reply:
agree, I'll update that.